IN-CORE/incore-playbook
# Test publish gh action

The main change is in publish github action:
- There is no need to `npm install` or `npm build` before publish
- But we need to properly set the .npmrc to point to github repo, plus the token

There shouldn't be any change need in the package.json. My IDE is having some reformatting setting; I can revert that part given more time.

To Test:
- See the successful github action here https://github.com/IN-CORE/incore-playbook/actions/runs/7994958276/job/21834199591
- See there is the 0.0.1 version published https://github.com/IN-CORE/incore-playbook/pkgs/npm/incore-playbook/181885553
- Install 0.0.1 with either of the testbed playbooks (e.g. Galveston-Playbook) and run `npm run start: dev`
```
"dependencies": {
		"@in-core/incore-playbook": "npm:@in-core/incore-playbook@^0.0.1",
		"@material-ui/core": "^4.11.0",
		...
```
